### DISCUSSION TITLE
Why are trailing slashes appended to the URL on refresh only for certain routes?

### DISCUSSION THREAD

**Discussion by Susccy** [nuxt/nuxt#10237] [Questions (Nuxt 2)]
I have tested this with a sample app in all 3 rendering modes. This app has 3 pages:

1. an index page at `example.com`
2. a sample page at `example.com/sample`
3. a dynamic page at `example.com/dynamic/_id`

When refreshing these pages in different rendering modes, in some cases a trailing slash is appended and in others not. These are the results:

**SSR** (default mode):
1. no trailing slash
2. no trailing slash
3. no trailing slash

**SSG** (`target: "static"`):
1. no trailing slash
2. trailing slash
3. trailing slash

**CSR** (`target: "static", ssr: false`):
1. no trailing slash
2. trailing slash
3. no trailing slash

What's the matter with these inconsistencies? Isn't this kinda bad for SEO in terms of duplicate pages?

**Comment by MathanKA** [comment]
Can someone know solution or root cause for this issue?

**Comment by monty-nz** [comment]
Bump

Have tried the following, but that causes an error page when accessing a page with a trailing slash. I just want it to remove the trailing slash... not present an error page
`  router: {
    trailingSlash: false,
  },`
